View/DataContexts/SearchDataContext.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Controller.DbControllers;
+using Model.Annotations;
+using Model.Data;
+using Model.DbModels;
+
+namespace View.DataContexts
+{
+    class SearchDataContext : INotifyPropertyChanged
+    {
+        private static SearchDataContext _instance;
+        public static SearchDataContext Instance
+        {
+            get
+            {
+                if (_instance == null)
+                   _instance = new SearchDataContext();
+                return _instance;
+            }
+        }
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public List<string> SearchTerms { get; set; } = new List<string>();
+        
+        public List<SongInfo> SearchSongs => SongInfo.ConvertSongListToSongInfo(SearchSongsOnString());
+        
+        public List<Playlist> SearchPlaylists => SearchPlaylistsOnStrings();
+
+        [NotifyPropertyChangedInvocator]
+        public void OnPropertyChanged([CallerMemberName] string propertyName = null)
+        {
+            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+        }
+
+        private List<Song> SearchSongsOnString()
```

Comment:
Deze functie kon niet vervangen worden met een lamba achter de property?

Reply:
denk niet, hij wou dat niet doen

Reply:
Hmm, want om nou voor alle props een functie te maken...

Reply:
Het werkt voor nu, later kunnen we het nog aanpassen, maar we moeten nu ook even door

Reply:
zou als het goed is nu moeten doen